docs/theory/UseCases.md

```diff
@@ -42,3 +40,55 @@ the first wrap key that is in turned encrypted with the second wrap key. The
 first wrap key is stored on a removable token device, e.g. YubiKey, and the
 second wrap key is sealed in a TPM NVRAM slot. For a system to boot it must
 have launched with the correct firmware and the token must be present.
+
+## Cross-links to publications mentioning TrenchBoot
```

Comment:
I think this would fit better on separate page. Please rename `Events` in main menu to `Events and publications` and create a new file in that category.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/43/commits/b26596a8a53cedb97dd3c484bd77602944f42fcb#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R14-R17